packages/dds/sequence/src/intervalCollection.ts

```diff
@@ -671,6 +705,7 @@ export class IntervalCollection<TInterval extends ISerializableInterval> extends
         this.localCollection = new LocalIntervalCollection<TInterval>(client, label, this.helpers);
         if (this.savedSerializedIntervals) {
             for (const serializedInterval of this.savedSerializedIntervals) {
```

Comment:
i really forgot how wierd this code was, but it looks like this does actually get called in both standalone, and merge tree cases. in most places attach graph is a one time operation where we attach something to the object graph, and its never called again